pyproject.toml

```diff
@@ -61,6 +61,7 @@ strict = [
     "phonopy==2.16.2",
     "seekpath==2.0.1",
     "numpy",
+    "mp-api>=0.27.5"
```

Comment:
Can this one be pinned to a specific version (e.g., `==`) as it makes the testing environment reproducible.

Reply:
Sure thing -- fair warning, this package is still being updated quite significantly until its "official release". For the functionality we're relying on here I expect it to be stable, but we'll probably want to keep this updated to the latest version fairly regularly.

Reply:
Thanks for the heads up. Hopefully dependabot should keep this in check.